viamonvif/service.go

```diff
@@ -522,6 +530,54 @@ func createCameraConfig(name string, attributes viamrtsp.Config) (resource.Confi
 	}, nil
 }
 
+func createPTZClientFromInfo(l CameraInfo, discoveryDependencyName string, logger logging.Logger) ([]resource.Config, error) {
+	ptzConfigs := []resource.Config{}
+	for index, u := range l.PTZEndpoints {
+		logger.Debugf("PTZ URL:\t%s", u)
+
+		discDep := ""
```

Comment:
Do we want to inject the discovery dependency? There is no refresh hook like in the rtsp camera model.

Reply:
The `discovery_dep` is cosmetic here. 
It is saying "the addresses in this config rely on this discovery service"
We do this for the rtsp camera models, so that it would be good to mirror here.

We only include discovery_dep if the address uses mDNS host name.

Reply:
oh so it'd just be like emulating what the real resp would look like?